scripts/airport-codes-flow.py

```diff
@@ -4,7 +4,7 @@
 from dataflows import validate, update_resource, add_metadata, dump_to_path
 
 
-def readme(fpath='README.md'):
+def readme(fpath='../README.md'):
```

Comment:
this probably didn't work and that's why `readme` property is null above. But we don't need this to be in the datapackage.json

Reply:
It should work now I have tested it locally, actually we have to leave it like it was before:
`def readme(fpath='README.md'):`